packages/babel-core/register.js

```diff
@@ -1,3 +0,0 @@
-/* eslint max-len: 0 */
-// TODO: eventually deprecate this console.trace("use the `babel-register` package instead of `babel-core/register`");
-module.exports = require("babel-register");
```

Comment:
I think there a lot of user relying on this.

Could we emit the warning for a period of time? Could remove this in a minor release later on.

([first link for "babel mocha" on Google](http://jamesknelson.com/testing-in-es6-with-mocha-and-babel-6/))